pom.xml

```diff
@@ -64,6 +64,10 @@
             <artifactId>mapstruct</artifactId>
             <version>${org.mapstruct.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
```

Comment:
duplicate dependency
its already in `spring-boot-starter-web` dependency
please remove it